gcp/appengine/frontend_handlers.py

```diff
@@ -748,14 +748,17 @@ def link_to_deps_dev(package, ecosystem):
 def display_severity_rating(severity: dict) -> str:
   """Return base score and rating of the severity."""
   severity_base_score, severity_rating = calculate_severity_details(severity)
+  if severity_base_score is None:
+    return "Invalid Severity Rating"
```

Comment:
nit: use single quotes
```suggestion
    return 'Invalid Severity Rating'
```